packages/browser/src/extensions/surveys.tsx

```diff
@@ -386,7 +386,7 @@ export class SurveyManager {
         if (!flagKey) {
             return true
         }
-        return !!this._posthog.featureFlags.isFeatureEnabled(flagKey)
+        return !!this._posthog.featureFlags.isFeatureEnabled(flagKey, { send_event: false })
```

Comment:
that will stop for all flags, see comment [here](https://posthog.slack.com/archives/C07QD3LT8U9/p1750057416504189?thread_ts=1749823234.899909&cid=C07QD3LT8U9).
we should only set that to our own internal flags only

Reply:
oops yeah, in my head I was like "let's not call this for all survey-related feature flags" but makes sense. changed now!